modules/core/src/main/java/org/apache/ignite/lang/ErrorGroups.java

```diff
@@ -390,4 +390,44 @@ public static class NodeConfiguration {
          */
         public static final int CONFIG_WRITE_ERR = NODE_CONFIGURATION_ERR_GROUP.registerErrorCode(3);
     }
+
+    /**
+     * Code deployment error group.
+     */
+    public static class CodeDeployment {
+        /**
+         * Code deployment error group.
+         */
+        public static final ErrorGroup CODE_DEPLOYMENT_ERR_GROUP = ErrorGroup.newGroup("CODEDEPLOY", 13);
+
+        /**
+         * Config read error.
+         */
+        public static final int UNDEPLOY_NOT_EXISTED_ERR = CODE_DEPLOYMENT_ERR_GROUP.registerErrorCode(1);
+
+        /**
+         * Unit duplicate error.
+         */
+        public static final int DUPLICATE_DEPLOY_UNIT_ERR = CODE_DEPLOYMENT_ERR_GROUP.registerErrorCode(2);
+
+        /**
+         * Access to not existed deployment unit.
+         */
+        public static final int NOT_EXISTED_UNIT_ERR = CODE_DEPLOYMENT_ERR_GROUP.registerErrorCode(3);
```

Comment:
```suggestion
        public static final int UNIT_NOT_FOUND_ERR = CODE_DEPLOYMENT_ERR_GROUP.registerErrorCode(3);
```

(to be consistent with `TABLE_NOT_FOUND_ERR`, `COLUMN_NOT_FOUND_ERR`)